aggregator/src/main/java/com/redhat/cloud/notifications/processors/email/aggregators/DailyEmailAggregationJob.java

```diff
@@ -63,7 +54,9 @@ public void processDailyEmail(ScheduledExecution se) {
         final LocalDateTime aggregateStarted = LocalDateTime.now();
 
         LOG.info(String.format("Running %s email aggregation for period (%s, %s)", EmailSubscriptionType.DAILY, startTime, endTime));
```

Comment:
I am a bit worried about the start/end time. This assumes, that the job always exactly starts at e.g. the full hour and 0 seconds. Now what if for whatever reasons the job starts one day at e.g. midnight and 0s 0ms and the next day at midnight +3s. We would then not aggregate the emails of the first 3s of the previous day. 

I have no idea if that is really a problem or more like a thought experiment.

Reply:
To be honest, I just took it from backend. It's the same there.

I think we should put it on our list. If it's a problem, it should still be a problem since we implemented it in notifications-backend.

Reply:
Yes, does not need to be changed now

Reply:
AFAIK it was a scheduled job of a running pod, but now it will be a new pod starting every day, so I think it's more subject to have delays as the startup time may change

Reply:
IIRC the start time and end time are taken from the "expected start date" and not the actual start date.

Reply:
So yes, the expected start date https://github.com/RedHatInsights/notifications-backend/pull/370/files#diff-ca42686170b3e81b0d58629ed3bea6644c4a638b80bbd7f8d85a43e57bdd5c6cR45